doc/btp_spec.txt

```diff
@@ -130,6 +130,14 @@ Commands and responses:
 
 		In case of an error, the error response will be returned.
 
+	Opcode 0x05 - Log message at IUT
+
+		Controller Index:	<non-controller>
+		Command parameters:	log message (variable)
```

Comment:
I would suggest to add "log_message_len" parameter, just like we have for other variable length parameters. We could think about log levels as well

Reply:
Sure. I've added a Log_Message_Len as in other commands.

For the use cases described above, I don't see a need for log levels. It could still be added after the log message if ever needed.